docs/readthedocs/source/doc/UserGuide/databricks.md

```diff
@@ -6,43 +6,62 @@ You can run BigDL program on the [Databricks](https://databricks.com/) cluster a
 ### **1. Create a Databricks Cluster**
 
 - Create either [AWS Databricks](https://docs.databricks.com/getting-started/try-databricks.html) workspace or [Azure Databricks](https://docs.microsoft.com/en-us/azure/azure-databricks/) workspace. 
-- Create a Databricks [clusters](https://docs.databricks.com/clusters/create.html) using the UI. Choose Databricks runtime version. This guide is tested on Runtime 7.3 (includes Apache Spark 3.0.1, Scala 2.12).
+- Create a Databricks [clusters](https://docs.databricks.com/clusters/create.html) using the UI. Choose Databricks runtime version. This guide is tested on Runtime 9.1 LTS (includes Apache Spark 3.1.2, Scala 2.12).
 
-### **2. Installing BigDL Python libraries**
+### 2. Download BigDL Libraries
 
-In the left pane, click **Clusters** and select your cluster.
+Download BigDL prebuilt release Python and Java libraries from [here](https://oss.sonatype.org/content/repositories/snapshots/com/intel/analytics/bigdl/bigdl-assembly-spark_3.1.2/2.1.0-SNAPSHOT/), scroll down to the bottom, download the **latest** release **bigdl-assembly-spark_3.1.2-2.1.0-*-fat-jars.zip**.
```

Comment:
Download BigDL package from [here]
download the **latest** -> choose the **latest**